Update README-template.md (#25)
IBM/MAX-Skeleton: README-template.md

To stop the Docker container, type `CTRL` + `C` in your terminal.

## Train this Model on Watson Machine Learning

> Remove this section if this model cannot be trained using custom data. Refer to https://github.ibm.com/CODAIT/max-model-training for details on how to enable a model for custom training.

> Whether it is fine-tuning the model with transfer learning or training from scratch on a custom dataset, retraining the model is a power feature. Please follow the steps listed under the [training readme](training/README.md) to retrain the model on [Watson Machine Learning](https://www.ibm.com/cloud/machine-learning), a deep-learning as a service (DLaaS) offering of [IBM Cloud](https://ibm.biz/Bdz2XM).

Reviewer: The first sentence is wordy. Maybe something like this?
```suggestion
This model supports both fine tuning with transfer learning and training from scratch on a custom model. Please follow the steps listed under the [training readme](training/README.md) to retrain the model on [Watson Machine Learning](https://www.ibm.com/cloud/machine-learning), a deep-learning as a service (DLaaS) offering of [IBM Cloud](https://ibm.biz/Bdz2XM).
```
